mmv1/third_party/terraform/acctest/bootstrap_test_utils.go

```diff
@@ -1107,6 +1107,78 @@ func BootstrapFirewallForDataprocSharedNetwork(t *testing.T, firewallName string
 	return firewall.Name
 }
 
+const SharedStoragePoolPrefix = "tf-bootstrap-storage-pool-"
+
+func BootstrapComputeStoragePool(t *testing.T, storagePoolName, storagePoolType string) string {
+	projectID := envvar.GetTestProjectFromEnv()
+	zone := envvar.GetTestZoneFromEnv()
+
+	storagePoolName = SharedStoragePoolPrefix + storagePoolName
```

Comment:
I think it would make sense to include the storage pool type here - or only allow storagePoolType and remove storagePoolName if we only need one of each type per region.

```suggestion
func BootstrapComputeStoragePool(t *testing.T, storagePoolType string) string {
	projectID := envvar.GetTestProjectFromEnv()
	zone := envvar.GetTestZoneFromEnv()

	storagePoolName := SharedStoragePoolPrefix + storagePoolType
```

Reply:
I'll update it to `storagePoolName = SharedStoragePoolPrefix + storagePoolType + storagePoolName` to make it easier to identify the type by reading the resource name. I'll also keep the storagePoolName, as we might need more than one storage pool per type as test cases expand.